corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -102,19 +102,25 @@ def render(self, name, value, attrs=None, choices=()):
         final_attrs = self.build_attrs(attrs, name=name)
 
         return format_html(
-            '<input{0} data-bind="'
-            '   questionsSelect: {1},'
-            '   value: \'{2}\','
-            '   optionsCaption: \' \''
-            '"/>',
+            """
+            <input{0} data-bind='
+               questionsSelect: {1},
+               value: "{2}",
+               optionsCaption: " "
+            '/>
+            """,
             flatatt(final_attrs),
             mark_safe(self.render_options(choices)),
             value
         )
 
     def render_options(self, choices):
+
+        def escape(literal):
+            return literal.replace('&', '&amp;').replace("'", "&#39;")
```

Comment:
is there a reason not to use something like `cgi.escape(literal, quote=True)` here?